handlers/recommendation.py

```diff
@@ -33,21 +33,27 @@ def get_recs(cls):
                 )
                 .order_by(Rec.score.desc())
             )
-            cls._recs = [x.to_dict() for x in query]
-            cls._last_updated = datetime.now()
+            if site:
+                query = query.join(
+                    Article, on=(Article.id == Rec.recommended_article)
+                ).where(Article.site == site)
```

Comment:
what happens here when site is None? the wcp frontend is still not sending that value yet, right?

Reply:
if site is None then there won't be any article filtering. Originally my plan was to enforce sending site and updating the wcp frontend. But maybe it's easier to just default to WCP and make sure our new clients are sending the site value?